roles/vg/vgcreate/README.md

```diff
@@ -0,0 +1,35 @@
+vgcreate
+=========
+vgcreate role creates a new volume group called VolumeGroupName using the block special device PhysicalDevicePath.
+
+Requirements
+------------
+If  PhysicalDevicePath  was not previously configured for LVM with pvcreate, the device will be initialized with the same default values used with pvcreate.  
+
+Role Variables
+--------------
+disk: disk or partition on which the pv is created.
```

Comment:
Yes, I'll give it a try with 3 disks just to be sure. Spoken to Sac about this too.